src/Console/Command/HelpCommand.php

```diff
@@ -376,6 +380,21 @@ protected function initialize(InputInterface $input, OutputInterface $output)
     }
 
     /**
+     * @return string|null
+     */
+    private static function getChangeLogFile()
+    {
+        $changelogFile = __DIR__.'/../../../CHANGELOG.md';
+        if (!is_file($changelogFile)) {
+            return null;
+        }
+
+        $changelogFile = realpath($changelogFile);
```

Comment:
In order to tell if it exists. It gives `false` if the path doesn't resolve, which is used on the next line.